### PR TITLE
Create allas-examples.md

### PR DESCRIPTION
**Pull request by kkmattil**

### CODE REVIEW

File: docs/data/Allas/allas-examples.md [hunk 1]
```diff
@@ -0,0 +1,513 @@
+# Examples for using Allas in CSC supercomputers 
+
+CSC supercomputers, Puhti and Mahti, do not provide permanent storage space for research data. 
+The data that needs to be for a longer time than just few weeks should be copied Allas object storage service.
```
Comment (Jukepoika) [inline docs/data/Allas/allas-examples.md:4]: "copied to Allas"

File: docs/data/Allas/allas-examples.md [hunk 2]
```diff
@@ -0,0 +1,513 @@
...
+Allas provides a platform that you can use to store your data as long as you CSC project is active. In addition 
+to storing, Allas can be used for transporting data between different servers and sharing data with outher users.
```
Comment (Jukepoika) [inline docs/data/Allas/allas-examples.md:6]: "with other users."

File: docs/data/Allas/allas-examples.md [hunk 3]
```diff
@@ -0,0 +1,513 @@
...
+
+*    [Allas user guide](./index.md)
+
+This tutorial provides four examples for using Allas in Puhti and Mahti. The eamples are based on interactively
```
Comment (Jukepoika) [inline docs/data/Allas/allas-examples.md:10]: "The examples are"

File: docs/data/Allas/allas-examples.md [hunk 4]
```diff
@@ -0,0 +1,513 @@
...
+The data that needs to be for a longer time than just few weeks should be copied Allas object storage service.
+Allas provides a platform that you can use to store your data as long as you CSC project is active. In addition 
+to storing, Allas can be used for transporting data between different servers and sharing data with outher users.
+
+*    [Allas user guide](./index.md)
+
+This tutorial provides four examples for using Allas in Puhti and Mahti. The eamples are based on interactively 
+executed commands and thus apply only for relatively small datasets (max some hundreds of GBs).
+
+1.   [The first example](#e1) uses the *a-commands* (a-put, a-get) for uploding data from Mahti to Allas and then downloading the data to Puhti.
```
Comment (Jukepoika) [inline docs/data/Allas/allas-examples.md:13]: "for uploading"

File: docs/data/Allas/allas-examples.md [hunk 5]
```diff
@@ -0,0 +1,513 @@
...
+members must visit the MyCSC and accept the terms of use for Allas before they can use the Allas 
+storage area.
+
+The default storage quota in Allas is 10 TB.  As this space is shared with all project members, it is  
+possible that the space is not sufficient. In that case, you should estimate how much space is needed 
+and request more space. The request should be sent to **servicedesk@csc.fi**.
+Please include in your quota request:
+
+   *    The ID/name of your project
+   *    The amount Allas space needed
```
Comment (Jukepoika) [inline docs/data/Allas/allas-examples.md:37]: "amount of"

File: docs/data/Allas/allas-examples.md [hunk 6]
```diff
@@ -0,0 +1,513 @@
...
+
+## A. Uploading data from Mahti to Allas
+
+The a-commands are Allas-specific tools that allow an easy start with Allas.  
+The a-commands pack, compress and move data automatically. This reduces the storage space needed but on
+the other hand makes the storage process bit slower. The a-commands are a good option for miscellaneous data 
+that compresses well and is mostly used in the CSC environment.
+
+In this example, I have the sub directory _genomes/zebrafish_ in the _scratch_ directory of my project in Mahti (/scratch/project_2001659). 
+The zebrafish directory  contains eight files listed below:
```
Comment (Jukepoika) [inline docs/data/Allas/allas-examples.md:54]: "directory contains" <- too much space between words

File: docs/data/Allas/allas-examples.md [hunk 7]
```diff
@@ -0,0 +1,513 @@
...
+Please wait.
+1) project_2000982     2) project_2001659     3) project_2000136      4) abort allas_conf
+Please choose a project by giving an item number from the list above: <b>2</b>
+
+allas connection configured successfully.
+Connection stays active for eight hours.
+</pre>
+
+`allas-conf` opens a connection to the specified Allas project for eight hours. If you want to start using 
+another project, you need to run `allas-conf` again. However, in one shell session, you can have only one Allas project active at a time.
```
Comment (Jukepoika) [inline docs/data/Allas/allas-examples.md:83]: "," is not needed after "session"

File: docs/data/Allas/allas-examples.md [hunk 8]
```diff
@@ -0,0 +1,513 @@
...
+Danio_rerio.GRCz10.91.rev.1.bt2  Danio_rerio.GRCz10.fa
+Danio_rerio.GRCz10.91.rev.2.bt2  Danio_rerio.GRCz10.fa.fai
+</pre>
+
+# Example 2: Using Allas with rclone <a name="e2"></a>
+
+## A. Uploading data with rclone
+
+Rclone is the power user tool for Allas. It is good in cases where the data does not compress much or 
+the data must be stored so that each file is stored as a separate object.
```
Comment (Jukepoika) [inline docs/data/Allas/allas-examples.md:216]: "data must stored as each file being a separate object"

File: docs/data/Allas/allas-examples.md [hunk 9]
```diff
@@ -0,0 +1,513 @@
...
+do
+rclone copy $f allas:2001659-uniref
+done
+```
+ 
+I can leave the session running in the background by pressing `Ctrl-a d`.
+
+Now, I can log out from Mahti, but the screen session remains active in the Mahti login node I use (in this case, _mahti-login1_).
+
+To connect to this session, I first connect to the mahti node where the screen session is running:
```
Comment (Jukepoika) [inline docs/data/Allas/allas-examples.md:397]: "mahti " -> "Mahti"

**Review by Jukepoika — COMMENTED** [review]
There is some unlogical use of pronouns we, I and you in the guide. It would be better to use only one of these (not I) or write in passive.

File: docs/data/Allas/allas-examples.md [hunk 10]
```diff
@@ -0,0 +1,513 @@
...
+a-put --nc --skip-filelist road_cameras/site_*/20*
+```
+This last option would store the data as 50 objects. Day-based objects for each camera might be the most practical 
+option for using the data later on but, as a downside, preprocessing the data into 10 * 5 * 365 = 18250 objects 
+probably takes quite a long time.
+
+Copying millions of files to Allas takes a long time regardless of the method. If you have started the `a-put` command inside a `screen` session, 
+you can detach from the virtual session by pressing `Ctrl-a-d` to log out from Puhti and leave the upload process running for days. 
+
+Once the _a-put_ command is finished, you can run `a-check` command to checkh if all the data objects have been created.
```
Comment (Jukepoika) [inline docs/data/Allas/allas-examples.md:495]: "checkh" --> "check"

**Review by attesillanpaa — APPROVED** [review]

**Pull request merged**